R/process_data_set_jordan.R

```diff
@@ -0,0 +1,33 @@
+library(tidyverse)
```

Comment:
This file should be in the `data-raw/` folder, which R ignores during the package build